uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Reminder.php

```diff
@@ -36,98 +36,129 @@ private static function _checkRelatedExtensions()
             self::$_relatedExtensions['appraisals'] = true;
         }
     }
-    
+
+    /**
+     * Gets the contacts starting with the given ids
+     *
+     * It also chains a call to the 'Email' api and puts the value in
+     * the 'email' property
+     *
+     * @param {Array} $contactIds
```

Comment:
I think the data type array is written as `array` rather than `{Array}` in the previous comments maybe ? Please have a look at `private static function _send`.